schema/schema.cc

```diff
@@ -1031,7 +1031,7 @@ sstring schema::get_create_statement(const schema_describe_helper& helper, bool
             else {
                 os << "    SELECT ";
                 for (auto& cdef : all_columns()) {
-                    if (cdef.is_view_virtual()) {
+                    if (cdef.is_view_virtual() || cdef.is_internal()) {
```

Comment:
I think we'll never have a materialized view with "internal columns", I don't know what this means.

By the way, it makes me wonder: What happens if you CREATE MATERIALIZED VIEW on a table that already has "internal columns"? We need to make sure that these columns are **not** copied into the view. We need to write a test for this - perhaps by trying to read explicitly the "internal" columns from the view (we allow explicit reads of a single column even if it's hidden).